### PR TITLE
Add staff_only property for creating guilds

### PR DESCRIPTION
**Pull request by BoryaGames**

### CODE REVIEW

File: pages/resources/guild.mdx [hunk 1]
```diff
@@ -922,6 +922,7 @@ Creates a new guild. Returns a [guild](#guild-object) object on success. Fires a
 | channels? ^2^                  | array[partial [channel](/resources/channel#channel-object) object] | Channels in the new guild                                                                                                  |
 | afk_channel_id?                | ?snowflake                                                         | The ID of the guild's AFK channel; this is where members in voice idle for longer than `afk_timeout` are moved             |
 | afk_timeout?                   | integer                                                            | The AFK timeout of the guild (one of 60, 300, 900, 1800, 3600, in seconds)                                                 |
+| staff_only? ^3^                | boolean                                                            | If new guild available for staff only.                                                                                     |
```
Comment (splatterxl) [inline pages/resources/guild.mdx:925]:
```suggestion
| staff_only? ^3^                | boolean                                                            | Whether the new guild will only be accessible for Discord employees                                                                                     |
```

Comment (dolfies) [reply pages/resources/guild.mdx:925]: Maybe also add an "(adds the [`INTERNAL_EMPLOYEE_ONLY` guild feature](#guild-feature))" at the end

File: pages/resources/guild.mdx [hunk 2]
```diff
@@ -930,6 +931,8 @@ Creates a new guild. Returns a [guild](#guild-object) object on success. Fires a
 
 ^2^ When specified, none of the default channels are created, and the `position` field is always ignored. Additionally, the `id` field within each channel object may be set to an integer placeholder, and will be replaced by the API upon consumption. Its purpose is to allow you to create `GUILD_CATEGORY` channels by setting the `parent_id` field on any children to the category's `id` field. Category channels must be listed before any children.
 
+^3^ Only available for Discord employees, otherwise it will return 403 (adds the [INTERNAL_EMPLOYEE_ONLY guild feature](/resources/guild#guild-features)).
```
Comment (splatterxl) [inline pages/resources/guild.mdx:934]:
```suggestion
^3^ Adds the `INTERNAL_EMPLOYEE_ONLY` flag, making the server only available for Discord employees.
```

**Review by splatterxl — CHANGES_REQUESTED** [review]
just wording issues

**Review by splatterxl — APPROVED** [review]

File: pages/resources/guild.mdx [hunk 3]
```diff
@@ -930,6 +931,8 @@ Creates a new guild. Returns a [guild](#guild-object) object on success. Fires a
 
 ^2^ When specified, none of the default channels are created, and the `position` field is always ignored. Additionally, the `id` field within each channel object may be set to an integer placeholder, and will be replaced by the API upon consumption. Its purpose is to allow you to create `GUILD_CATEGORY` channels by setting the `parent_id` field on any children to the category's `id` field. Category channels must be listed before any children.
 
+^3^ Adds the `INTERNAL_EMPLOYEE_ONLY` flag, making the server only available for Discord employees.
```
Comment (dolfies) [inline pages/resources/guild.mdx:934]:
```suggestion
^3^ Adds the [`INTERNAL_EMPLOYEE_ONLY` guild feature](#guild-features), making the server only available for Discord employees. Only settable by Discord employees.
```

**Review by dolfies — CHANGES_REQUESTED** [review]

**Review by dolfies — APPROVED** [review]

**Pull request merged**